portal/src/main/webapp/css/patient-view/clinical-attributes.css

```diff
@@ -120,15 +120,15 @@
     content: " years old";
 }
 /* attributes with special colors */
-.clinical-attribute[attr-id="OS_STATUS"][attr-value="DECEASED"],
-.clinical-attribute[attr-id="OS_STATUS"][attr-value="DEAD"],
+.clinical-attribute[attr-id="OS_STATUS"][attr-value="1:DECEASED"],
```

Comment:
Are we not generalizing for other survival types here (admittedly I don't know where this file is used).

Reply:
Same here, I think this file is also outdated. We are not using this anymore. I just want to keep the consistency between all the files, so I also changed this.